tests/e2e/pageObjects/RecipePage.js

```diff
@@ -0,0 +1,72 @@
+const config = require("../../cucumber.conf.js")
+const { format } = require('util')
+
+class RecipePage {
+    constructor() {
+        this.createRecipeDropdown = "//i[@class='fas fa-fw fa-plus fa-lg']"
```

Comment:
is it possible to get the xpath of this element with a better id or class name, does the parent component have some useful name that we can use?
```suggestion
        this.createRecipeDropdown = "//i[@class='fas fa-fw fa-plus fa-lg']"
```

Reply:
the parent component have better id and class but not unique